src/docs/environment/index.mdx

```diff
@@ -89,23 +89,55 @@ computer is actually compiling Python!
 make setup-pyenv
 ```
 
+fish users will need to manually set some of the environment variables. This only needs to be done
+once.
+
+```shell {tabTitle: Fish}
+set -Ux PYENV_ROOT $HOME/.pyenv
+# fish >=3.2.0
+fish_add_path $PYENV_ROOT/bin
+# fish <3.2.0
+set -U fish_user_paths $PYENV_ROOT/bin $fish_user_paths
+```
+
 Once that's done, your shell needs to be reloaded. You can either reload it in-place, or close your terminal and start it
 again and cd into sentry. To reload it, run:
 
-```shell
+```shell {tabTitle: Bash}
+exec "$SHELL"
+```
+
+```shell {tabTitle: Zsh}
 exec "$SHELL"
 ```
 
+```shell {tabTitle: Fish}
+exec $SHELL
+```
```

Comment:
This change was made in a few other additional places in the PR.

It's less code overall to just mention fish's differences in text, but it's also additional
required reading for the end user. Stuffing these into code blocks provides a more streamlined
experience since their selected languages sync across an entire page, meaning that once a user
decides to view the fish instructions in one code block, the others will switch to displaying
fish as well.